ffi/src/error.rs

```diff
@@ -92,6 +105,8 @@ pub mod ffi {
         AccessDenied,
         #[error("Incorrect rust enum type")]
         IncorrectEnumType,
+        #[error("Clipboard error")]
+        ClipboardError,
```

Comment:
naming: Do not use the "Error" suffix in variant names, this is redundant with the enum name

Reply:
updated